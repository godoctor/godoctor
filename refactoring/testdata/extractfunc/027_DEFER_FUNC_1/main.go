//<<<<<extract,7,2,9,2,Foo,fail
package main

import "fmt"

func main() {
	for i := 0; i < 4; i++ {
		defer fmt.Print(i)
	}
}
