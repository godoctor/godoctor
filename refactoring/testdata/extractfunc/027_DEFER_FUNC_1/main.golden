//<<<<<extract,7,2,9,2,Foo,fail
package main

import "fmt"

func main() {
	b()
}

func b() {
	for i := 0; i < 4; i++ {
		defer fmt.Print(i)
	}
}
